Todo-Redux/src/js/modules/prelogin/actions.js

```diff
@@ -0,0 +1,32 @@
+const uid = () => Math.random().toString(34).slice(2);
+
+export function addTodo(text) {
+  return {
+    type: 'ADD_TODO',
+    payload: {
+      id: uid(),
+      status: false,
+      text: text
+    }
+  };
+}
+
+export function searchTodo(text) {
+  return {
+    type: 'SEARCH_TODO',
```

Comment:
create a separate CONSTANTS file for actions, action creator should refer action from there

Reply:
Added constants.js file